GVFS/GVFS.Common/Maintenance/PackfileMaintenanceStep.cs

```diff
@@ -154,14 +167,28 @@ protected override void PerformMaintenance()
                 metadata.Add(nameof(afterSize), afterSize);
                 metadata.Add("ExpireOutput", expireResult.Output);
                 metadata.Add("ExpireErrors", expireResult.Errors);
+                metadata.Add("verifyAfterExpire", verifyAfterExpire.ExitCode);
```

Comment:
nit: capitalize for consistency, and maybe name this and `VerifyAfterRepack` `Verify*ExitCode`